doc/holochain_101/src/overview.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+Depending on whether your interest is specific, or general, you may wish to read it front to back, or skip to specific sections of the book. If the summary of a section describes you, then it's a section for you!
+
+### Planning a dApp
+**Readers:** You can't put the horse before the cart. First things first:  understanding the landscape of decentralized apps. What are the mechanics of a functioning decentralized app? You have an idea of what you want to build, and you need to get a grasp on what it's going to take. You need to know what your blind spots are, and what are the common pitfalls. You need to know which of your assumptions to hold on to, and which to let go. 
+
+**Writing:** General Audience
+
+### Building Holochain Apps
+
+**Readers:** Whether you're a seasoned developer, or just starting out, you're in it to write code. You've either got an app project (or five) on the go, or you're in it to experiment and test the limits. You need to know what you need to know. You want to talk Holochain's language. You're an intrepid explorer of technical documentation.
+
+**Writing:** Technical, Explanatory & How-to
+
+### Going Live with Holochain Apps
+
+**Readers:** You're involved in the conception, development, or design of a Holochain app, and you've got to know how to get your app out into the world, into the hands of the people who need it. You've got questions like "how do updates to the app work?" "how do I track performance of the app?" "what are best practices for security?"
+
+**Writing:** General Audience
+
+### Extending the Holochain Platform
+
+**Readers:** You want to look at Holochain itself, not what you can build with it, and see what you can do, or contribute. You've got ideas for Holochain and got skills to pull them off. You're reading the Holochain source code, or source documentation. Maybe you want to enable app development in a whole other language not available yet, or maybe to run Holochain on a device or platform not supported yet. You can make sense of terse technical language, and direct yourself well.
+
```

Comment:
"You want to look at Holochain itself, not what you can build with it, and see what you can do, or contribute." -- rephrase?  I get lost by 'not what you can build with it'.  Is it, '...not what you can build with it, but to see what you can do or contribute'?